maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationServiceTest.java

```diff
@@ -284,14 +293,16 @@ void givenMagCourt_whenCreateIsInvokedAndExceptionThrownInGetSummary_thenRollbac
         when(assessmentSummaryService.getSummary(any(HardshipReviewDTO.class), eq(CourtType.MAGISTRATE)))
                 .thenThrow(new APIClientException());
 
+        when(hardshipMapper.getUserActionDTO(any(), any()))
+            .thenReturn(UserActionDTO.builder().username("mock-u").build());
+
         assertThatThrownBy(() -> orchestrationService.create(workflowRequest))
                 .isInstanceOf(MaatOrchestrationException.class);
         Mockito.verify(hardshipService, times(1)).rollback(any());
```

Comment:
More opportunities for some cleanup - if we're mocking something then verifying it too is redundant, as if the mock isn't called an error will be thrown regardless.

Reply:
Are you referring here to all of the instances of `Mocito.verify` where we've set up the mock with a `when`?

Reply:
Yes, that's right - verifying a mock is redundant

Reply:
Gotcha. All nine occurrences in this test file where we currently call `Mockito.verify` are to verify that the `rollback` method on the `HardshipService` has been called and I don't think we're explicitly setting that method up anywhere, so I would expect that these calls to verify the method are valid.

I have a commit ready to go for this but just want to double-check before I push it.

Reply:
That's fine, it's really just the instances where we're mocking a call and then also verifying it.